src/sorcha/utilities/sorchaArguments.py

```diff
@@ -0,0 +1,83 @@
+import numpy as np
+from dataclasses import dataclass
+import time
+from os import path
+
+
+@dataclass
+class sorchaArguments:
+    """Data class for holding runtime arguments"""
+
+    paramsinput: str = ""
+    """path to file with input objects"""
+    orbinfile: str = ""
+    """path to file with input object orbits"""
+    oifoutput: str = ""
+    """path the OIF output file"""
+    configfile: str = ""
+    """path to the config.ini file"""
+    outpath: str = ""
+    """path where data should be output"""
+    outfilestem: str = ""
+    """file system for output"""
+
+    verbose: bool = False
+    """logger verbosity"""
+
+    makeTemporaryEphemerisDatabase: bool = False
+    """whether or not to make ephemeris database"""
+    readTemporaryEphemerisDatabase = False
+    """path to ephemeris database or `False`"""
```

Comment:
I don't think we should change it, but this doesn't seem like an ideal variable name.